frontend/src/pages/user/LessonAnswer.tsx

```diff
@@ -0,0 +1,144 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react'
+import { useNavigate, useSearchParams } from "react-router-dom";
+import { Button, Col, Row, Stack } from 'react-bootstrap'
+import { Category } from '../../interfaces/category';
+import { Question } from '../../interfaces/question';
+import { Lesson } from '../../interfaces/lesson'
+import { AnswerCreation } from '../../interfaces/answer';
+import { getLesson } from '../../apiClient/lessonService';
+import { getCategory } from '../../apiClient/categoryService';
+import { listQuestionsByCategory } from '../../apiClient/questionService';
+import { listChoicesByQuestion } from '../../apiClient/choiceService';
+import { createAnswer } from '../../apiClient/answerService';
+import routes from '../../constants/routes';
+
+function LessonAnswer() {
+  const navigate = useNavigate()
+  let [searchParams] = useSearchParams();
+  const lessonId = Number(searchParams.get("id"));
+
+  const [lesson, setLesson] = useState<Lesson>({
+    id: 0,
+    user: 0,
+    category: 0,
```

Comment:
rename to a more clearer description
```suggestion
    userId: 0,
    categoryId: 0,
```